docs/milestone-1.md

```diff
@@ -0,0 +1,17 @@
+# Milestone 1
+## Context
+The milestone’s target is to be able to demonstrate the automatic provisioning
+of a Storage Tenant to a VMware Cloud Director tenant organizations using
+vCloud Director UI.
+
+## Acceptance criteria
+The acceptance criteria for the milestone are:
+-   The CI produces a docker image and stores it in the Scality registry
+-   The image can be run and contain the OSE
+-   The container can be integrated with a vault instance
```

Comment:
What do you mean by integrate? is it just a matter of being able to communicate with a vault API (if vault works that way, my knowledge of it is limited :D ) 
Or does it needs to run along with vault?

Because the wording `the container can be integrated` I understand that the container itself have some kind of integration with vault, like as they would run kind of together, rather than saying that the service can be integrated vault.

Worth noting that maybe that's just me misunderstanding things.

Reply:
You're right 👍 

By integrated I mean, the Object Storage Extention(OSE) should be able to communicate with a vault API.

I updated the acceptance criteria, is it better?